### PR TITLE
Added Volume of Torus

### PR DESCRIPTION
**Pull request by aladin002dz**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in their comments that points to Wikipedia or another similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Maths/test/Volume.test.js [hunk 1]
```diff
@@ -44,3 +44,9 @@ test('Testing on volHemisphere', () => {
   const volHemisphere = volume.volHemisphere(4.0)
   expect(volHemisphere).toBe(134.0412865531645)
 })
+
+test('Testing on volTorus', () => {
+  const volTorus = volume.volTorus(10.0, 5.0)
+  console.log(volTorus)
```
Comment (appgurueu) [inline Maths/test/Volume.test.js:50]: This shouldn't be here.

File: Maths/Volume.js [hunk 2]
```diff
@@ -112,6 +112,17 @@ const volHemisphere = (radius) => {
   return (2.0 * Math.PI * radius ** 3) / 3.0
 }
 
+/**
+ * Calculate the volume for a Torus
+ * Reference: https://www.mathsisfun.com/geometry/torus.html
+ * return 2 * (Math.PI^2) * R * (r^2)
```
Comment (appgurueu) [inline Maths/Volume.js:118]: `@return`

File: Maths/Volume.js [hunk 3]
```diff
@@ -112,6 +112,17 @@ const volHemisphere = (radius) => {
   return (2.0 * Math.PI * radius ** 3) / 3.0
 }
 
+/**
+ * Calculate the volume for a Torus
+ * Reference: https://www.mathsisfun.com/geometry/torus.html
```
Comment (appgurueu) [inline Maths/Volume.js:117]: `@see`

**Review by appgurueu — COMMENTED** [review]
Code-wise trivially fine. Not at all algorithmically interesting though. I'm not sure if this warrants inclusion. @raklaptudirm thoughts?

**Comment by raklaptudirm** [comment]
Its just the code implementation of a mathematical formula. The `volume.js` file already has too many examples for this to warrant inclusion.

**Pull request closed without merging**